modules/gateway-task/main.tf

```diff
@@ -105,7 +105,7 @@ resource "aws_ecs_task_definition" "this" {
               initProcessEnabled = true
             }
             healthCheck = {
-              command  = ["CMD-SHELL", "curl localhost:10000/consul-ecs/health"] # consul-ecs-control-plane exposes a listener on 10000 to indicate it's readiness
+              command  = ["CMD-SHELL", "curl -f localhost:10000/consul-ecs/health || exit 1"] # consul-ecs-control-plane exposes a listener on 10000 to indicate it's readiness
```

Comment:
the `|| 1` isn't redundant after adding the curl flag `-f`?

Reply:
Yes it is. I can remove the same.